run.yml

```diff
@@ -63,12 +63,17 @@
   - name: Copy agent iso to /var/www/html
     become: yes
     command: cp billi/agent.x86_64.iso /var/www/html/{{ cluster }}.iso
+  - name: Get all users
+    getent:
```

Comment:
that requires root access, let's figure a simpler way to know we are within a container